### ISSUE TITLE
Country selection list is incomplete 4.1.0 + Postgresql 9.6

### ISSUE THREAD

**Issue by pongraczi** [laravel-enso/enso#323]
This is  a **bug**.

### Prerequisites
* [x] Are you running the latest version?
* [x] Are you reporting to the correct repository?
(enso is made of many specialized packages: https://github.com/laravel-enso)
* [ ] Did you check the documentation?
* [ ] Did you perform a cursory search?
* PHP 7.4, Postgresql 9.6, fresh install from git

### Description

Country selection list is incomplete, even countries table is filled.  
![Kijelölés_385](https://user-images.githubusercontent.com/5891983/89547982-bd90ac80-d806-11ea-865b-b57f630a4b93.png)

### Steps to Reproduce
1. Create new Address
2. You will get this, where Réunion already selected:  
![Kijelölés_384](https://user-images.githubusercontent.com/5891983/89548063-d9944e00-d806-11ea-9bcf-c1192d7eafcf.png)
3. Try to find Hungary -> no result, the list ends like this:
![Kijelölés_385](https://user-images.githubusercontent.com/5891983/89548151-f466c280-d806-11ea-8be5-26c4335726c3.png)

### Expected behavior
Full list of all countries.  

### Actual behavior
As shown above.

**Comment by pongraczi** [comment]
Inspection inside the browser shows this:  
![Kijelölés_386](https://user-images.githubusercontent.com/5891983/89550057-6dffb000-d809-11ea-81d9-64fde957ecc8.png)  
So, the list ends with Åland Islands.

* I checked the database query in debug mode and I found that, there is a pagination settings to 100. `"paginate":"100"`
  * /api\/core\/countries\/options?customParams=%7B%7D&paginate=100&params=%7B%7D&pivotParams=%7B%7D&query=&searchMode=full&trackBy=id&value=184"
* I just count the numbers of existing countries, there are 100 elements, so, bingo.

My question, is it a user settings of global one to paginate this list? In other words, could the user change this behaviour?

**Comment by pongraczi** [comment]
Same effect on the laravel-enso.com site, too.

**Comment by aocneanu** [comment]
Having the full option list in the front-end would pretty much cancel the need for server side load.

The select displays only the first 100 entries initially, but you can type in the searchbox anything and the server will return the records the match (first 100).

Or disable the server side for this input and send all the options when the form loads from the form builder.

**Comment by pongraczi** [comment]
Thanks, maybe it is only me, but for me this function seems a little bit inconsistent.  
Here I started typing on my installation: united, expecting some results, but nothing appears.  
![Kijelölés_387](https://user-images.githubusercontent.com/5891983/89618978-6b469e80-d88d-11ea-9d70-8c0745e8c6a5.png)

Meanwhile it now works on your demo site.  

My infrastructure is:  
Internet ---- apache2 reverse proxy --- dedicated enso server with nginx  --- dedicated Postgresql database server

**Comment by pongraczi** [comment]
Ok, I got it.  
The query is case sensitive and that is the reason I got no results when I started to type like this: united.  
But when I started to type with Unite... I got the expected result.  

As I checked the database server log (debug), I found this (only a snippet):
```
where ((\"name\"::text LIKE ''%fran%'') or (\"iso_3166_3\"::text LIKE ''%fran%'')) 
```
and the LIKE is definitely case sensitive.

Ok, it seems the "problem" solved like this: there is no problem, but the input is case sensitive **in case of Postgresql backend**.  

I guess it is related to Eloquent which translate the query for specific databases.
